Makefile

```diff
@@ -80,6 +80,9 @@ benchmarks-messages:
 benchmarks-schemas:
 	./scripts/run_benchmark.sh -p schemas
 
+benchmarks-stateful-message-storage:
```

Comment:
I think `message` seems extra in the name and it might confuse people with regular message pallet. I prefer `stateful-storage` 

what do you think?

Reply:
That was the result of the last poll I posted in Slack. But I agree, it's quite a mouthful. If we're in agreement that we prefer `stateful-storage`, I'll make the change.

Reply:
Renamed.

Reply:
Renamed.